.github/test-suite/determine_gosop_branch.sh

```diff
@@ -0,0 +1,7 @@
+VERSION=$(awk '/^module github.com\/ProtonMail\/gopenpgp\/v[0-9]+/ {print $NF}' gopenpgp/go.mod | awk -F'v' '{print $2}')
+
+if [ "$VERSION" -eq 3 ]; then
+  echo "gosop-gopenpgp-v3"
+else
+  echo "main"
+fi
```

Comment:
I think this needs to be updated, right?

Reply:
This script determines which gosop to take. In the gosop repo `main` is based on v2 and `gosop-gopenpgp-v3` on v3. Do we want to switch there as well?

Reply:
done.